### ISSUE TITLE
Custom Domain setting

### ISSUE THREAD

**Issue by MironChen** [abhijithvijayan/ghost-on-github-pages#32]
Hi, it's me again.

According to your `deploy.sh` , I can only deploy to `username.github.io `. However, as long as I deploy for the next time, even though I have set a custom domain in Github, it will insist on `username.github.io `.

I wonder what should I change.

Thanks a lot.

**Comment by pavlenex** [comment]
Here's what I tried and it worked for me.
Instead github pages, yourdomain.github.io I created a regular repostirory where I deployed everything. Then I simply connected to Netifly where I configured my DNS and it worked. I am checking if building process is automatic and if I have to do something more.

**Comment by pavlenex** [comment]
Yup can confirm netlify + this works amazing with custom domain and there's no configuration needed regarding building, it's just a few clicks defaults.

**Comment by abhijithvijayan** [comment]
I am sorry that I missed the notifications for the issue(was set busy in github status). Will take a look at this.

**Comment by abhijithvijayan** [comment]
Yeah, I have migrated from GitHub page to Netlify some time ago. Never looked back.

The whole aim of the project is to deploy the dynamic blog as a static one and host without charges(with custom domain and other features).

Like @pavlenex said, Use Nelify to deploy the static page.

- Create a regular repo(with CLI for easiness `npx create-remote-repo blog`)
- Deploy the content to the repository
- Connect Netlify to the repository
- Deploy with Neltify
- Setup custom domain if needed
- Voila :)

**Comment by abhijithvijayan** [comment]
With netlify I see this error 
`Deployed filenames cannot contain # or ? characters` for the `'assets/built/custom.css?v=536fdc0854.css'` entry

@pavlenex Did you have any issues while building with Netlify?

**Comment by pavlenex** [comment]
Hey @abhijithvijayan It didn't. It worked well for me. Worked exactly as you described [above](https://github.com/abhijithvijayan/ghost-on-github-pages/issues/32#issuecomment-636375614).  Unfortunately I'm not sure how I can help besides that, quite new to all this. Let me know if there's something I can do to help reproduce though.

**Comment by abhijithvijayan** [comment]
Never mind. It's the netlify that stopping the build process.

`?version` query param is added for cache busting.

Probably have to figure out another way to make it work with Netlify.
